File: src/main/java/no/nav/veilarboppfolging/controller/request/Innsatsgruppe.java
package no.nav.veilarboppfolging.controller.request;

public enum Innsatsgruppe {
    STANDARD_INNSATS("IKVAL"),
    SITUASJONSBESTEMT_INNSATS("BFORM"),
    BEHOV_FOR_ARBEIDSEVNEVURDERING("BKART");

    private String innsatsgruppeKode;

    Innsatsgruppe(String innsatsgruppeKode) {
        this.innsatsgruppeKode = innsatsgruppeKode;
    }

    public String getKode() {
        return innsatsgruppeKode;
    }
}

File: src/main/kotlin/no/nav/veilarboppfolging/controller/request/Innsatsgruppe.kt
package no.nav.veilarboppfolging.controller.request

enum class Innsatsgruppe(innsatsgruppeKode: String) {
    STANDARD_INNSATS("IKVAL"),
    SITUASJONSBESTEMT_INNSATS("BFORM"),
    BEHOV_FOR_ARBEIDSEVNEVURDERING("BKART");

    private val kode: String?

    init {
        this.kode = innsatsgruppeKode
    }
}
